docs/en/ingest-management/elastic-agent/configuration/elastic-agent-configuration.asciidoc

```diff
@@ -21,6 +21,32 @@ changes after installation, you must modify the installed file.
 
 For installation details, refer to <<install-standalone-elastic-agent>>.
 
+Alternatively, you can put input configurations in YAML files into the
+folder `{path.config}/inputs.d` to separate your configuration into
+multiple smaller files.
+Glob that has to match the path is `{path.config}/inputs.d/*.yml`.
```

Comment:
Is there something missing here? This sentence doesn't make sense in the context of this paragraph.

Reply:
Do users need to configure settings in the elastic-agent.yml file to enable use of the inputs.d folder?

Reply:
It was just a leftover from one of my drafts. Thanks.

> Do users need to configure settings in the elastic-agent.yml file to enable use of the inputs.d folder?

 No, Agent automatically picks up files from the inputs.d folder.